app/portainer/views/endpoints/edit/endpointController.js

```diff
@@ -314,89 +319,130 @@ function EndpointController(
     $scope.endpoint.ManagementInfo['DNS Suffix'] = '-';
   }
 
-  function buildLinuxStandaloneCommand(agentVersion, agentShortVersion, edgeId, edgeKey, allowSelfSignedCerts) {
-    return `
-docker run -d \\
-    -v /var/run/docker.sock:/var/run/docker.sock \\
-    -v /var/lib/docker/volumes:/var/lib/docker/volumes \\
-    -v /:/host \\
-    -v portainer_agent_data:/data \\
-    --restart always \\
-    -e EDGE=1 \\
-    -e EDGE_ID=${edgeId} \\
-    -e EDGE_KEY=${edgeKey} \\
-    -e CAP_HOST_MANAGEMENT=1 \\
-    -e EDGE_INSECURE_POLL=${allowSelfSignedCerts ? 1 : 0} \\
-    --name portainer_edge_agent \\
-    portainer/agent:${agentVersion}`;
+  function buildEnvironmentSubCommand() {
+    let env = [];
```

Comment:
this variable is not necessary, you could simplify by doing sth like
```suggestion
    if ($scope.formValues.EnvVarSource === '') {
        return [];
    }
    
    return $scope.formValues.EnvVarSource.split(',')....
```